core/src/main/scala/besom/util/interpolator.scala

```diff
@@ -10,7 +10,7 @@ object interpolator:
 
   implicit final class PulumiInterpolationOps(sc: StringContext):
     def pulumi(args: Any*)(using Context): Output[String] =
```

Comment:
Shouldn't `(using Context)` be unnecessary now (here and for `p`)?

Reply:
It's WIP, I haven't completed the search for all `(using Context)` clauses in public APIs yet.